src/threading.c

```diff
@@ -461,9 +461,7 @@ JL_DLLEXPORT jl_gcframe_t **jl_autoinit_and_adopt_thread(void)
                             "       (this should not happen, please file a bug report)\n");
             exit(1);
         }
-        const char *image_path = jl_pathname_for_handle(handle);
-        jl_enter_threaded_region(); // This should maybe be behind a lock, but it's harmless if done twice
```

Comment:
Was this deletion intentional?

Reply:
Yes. It got moved into the jl_init_with_handle call (the threaded region maybe should still be here :\ but I dunno)

Reply:
Yeah it was specifically the thread region I was curious about - I'm OK with deleting it for now, we can re-visit soon with thread-safety